src/components/TeamHub.js

```diff
@@ -160,152 +163,110 @@ module.exports = React.createClass({
 
 		// Update the nodes
 		node = kudos
-			.selectAll('circle.node')
+			.selectAll('.node')
 			.data(nodes, (d) => { return d.id; })
 
-		node.transition()
-			// .style('fill', 'black')
-			// .attr('r', 20);
-			// .attr('r', (d) => { return d.children ? 45 : Math.sqrt(d.size) / 10; });
-			// .attr('r', (d) => { return d.children ? 30 : d.size*10; });
+		// This is the base of the radius used for circles/images
+		let r = 20;
 
 		// Enter any new nodes
-		let r = 20;
 		node
 			.enter()
-			.append('svg:circle')
+			.append('g')
+			.attr('class', 'node')
+			.call(force.drag)
+			.append('image')
 			.attr('class', (d) => {
-				let classes = ['node'];
-				d.username && classes.push(d.username);
-				return classes.join(' ');
+				return d.username || '';
+			})
+			.attr('xlink:href', (d) => {
+				if (!d.children && !d._children) {
+					return `../images/${d.username}.png`;
+				} else {
+					return `../images/empty.png`;
+				}
+			})
+			.attr('x', (d) => {
+				if (d.isRoot) return -r*3;
```

Comment:
Also, it might look cleaner to set the `attr`s using the object literal syntax (if http://stackoverflow.com/a/20822552/342886 is to be believed) (since the if conditions are repeating, maybe they can be refactored into a function or something? I'm just thinking out loud).

This isn't super important though, so it's up to you.